docs/troubleshoot/ze/troubleshoot-ze.md

```diff
@@ -2,6 +2,9 @@
 
 As a Zowe Explorer user, you may encounter problems when using Visual Studio Code extension functions. Review Zowe Explorer known issues and troubleshooting solutions here.
 
+**[--add troubleshooting note on how to clear user credentials from secure storage. Maybe document later for cli and ze; could be included in troubleshooting section if users are having an issuew/ their stored secrete creds, steps to clear cache and start fresh]**
```

Comment:
Should this TODO comment be addressed, or is it for a future PR?

Reply:
@t1m0thyj - This will be addressed -- it's a placeholder.